website/views.py

```diff
@@ -101,13 +101,19 @@ def find_bookmark_collection(user):
 
 @must_be_logged_in
 def dashboard(auth):
+    return redirect('/')
+
+
+@must_be_logged_in
+def my_projects(auth):
     user = auth.user
-    dashboard_folder = find_bookmark_collection(user)
-    dashboard_id = dashboard_folder._id
+    my_projects_folder = find_bookmark_collection(user)
```

Comment:
This should be `bookmark_collection` rather than `my_projects_folder`. Same with the similar variables.